wikihow2zim/scraper.py

```diff
@@ -29,9 +38,73 @@ def __init__(self, **kwargs):
         for option in options:
             setattr(self, option, kwargs[option])
 
+        if self.output_dir is None:
+            self.output_dir = pathlib.Path(os.getcwd()).joinpath("output")
+
+        pathlib.Path(self.output_dir).mkdir(parents=True, exist_ok=True)
+
         self.url = URLS[self.language]
 
+        # jinja2 environment setup
+        self.env = Environment(
+            loader=FileSystemLoader(str(self.templates_dir)), autoescape=True
+        )
+
+        # Set up the output zim file:
+        fpath = pathlib.Path(self.output_dir).joinpath("output.zim")
```

Comment:
Not much value in creating this variable just to use it once on next line.
You may want to keep that as an instance prop though if you want to display it later on in the process.

Reply:
@rgaudin The only reason I have it like that is that the code looks more readable to me. Otherwise the line is too long and if I try to have it in several lines it doesn't look good. Creating a single variable like this isn't particularly computationally expensive, but we can change it to be in one line if you think it's better. Either way is fine with me.